src/tm_devices/commands/gen_e3e9uu_lpdmso/saveonevent.py

```diff
@@ -9,7 +9,8 @@
 
 Commands and Queries:
     ```
-    - SAVEONEVent:FILEDest ? <Qstring>
+    - SAVEONEVent:FILEDest
```

Comment:
Is this command missing an argument?

Reply:
Actually write syntax should accept the argument and query syntax should not accept the arguments.